src/api/commands/itemimage/ItemImageRemoveCommand.ts

```diff
@@ -33,11 +33,11 @@ export class ItemImageRemoveCommand extends BaseCommand implements RpcCommandInt
      */
     @validate()
     public async execute( @request(RpcRequest) data: RpcRequest): Promise<void> {
-        if (data.params.length < 1) {
-            throw new MessageException('Requires arg: ItemImageId');
-        }
-        const itemImageModel = await this.itemImageService.findOne(data.params[0]);
-        const itemImage = itemImageModel.toJSON();
+        const itemImageId = data.params[0];
+        const itemImage = await this.itemImageService.findOne(itemImageId);
+
+        // find related itemInformation
+        const itemInformation = itemImage.related('ItemInformation').toJSON();
```

Comment:
findOne returns the relations by default, theres no need to call related('ItemInformation').